solr-ingest/src/main/java/edu/unc/lib/dl/data/ingest/solr/filter/SetDescriptiveMetadataFilter.java

```diff
@@ -97,23 +97,18 @@ public void filter(DocumentIndexingPackage dip) throws IndexingException {
     }
 
     private String getAlternativeTitle(DocumentIndexingPackage dip) throws FedoraException, IndexingException {
-        String title = null;
         Resource resc = dip.getContentObject().getResource();
 
         // Use dc:title as a default
         if (resc.hasProperty(DcElements.title)) {
             Statement dcTitle = resc.getProperty(DcElements.title);
             return dcTitle.getString();
-        }
-
-        // fall back to filename if one is present
-        if (title == null && resc.hasProperty(Ebucore.filename)) {
+        } else if (resc.hasProperty(Ebucore.filename)) { // fall back to filename if one is present
```

Comment:
Should we do empty checks before returning these values? `StringUtils.isBlank` catches null, blank, or all whitespace. If the title is present but empty, we'd want to fall back to the other options